spec/features/search_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'features_helper'
+
+feature 'Search' do
+  let(:user) { create(:user) }
+  let!(:question) { create(:question, title: 'Title', body: 'Sphinx, RSpec, Rails', user: user) }
+
+  before do
+    index
+    visit search_path
+  end
+
+  scenario 'Success, user implements search', js: true do
```

Comment:
Please, add specs for all search types.
You should also check the results (not only "no matches" case). Make your tests more meaningful.

Also add a special scenario: searching by full email address.